component/src/main/java/org/cbioportal/genome_nexus/component/annotation/NotationConverter.java

```diff
@@ -322,7 +328,13 @@ else if (ref.length() > 1 || var.length() > 1) {
     public List<String> genomicToEnsemblRestRegion(List<GenomicLocation> genomicLocations)
     {
         List<String> ensemblRestRegionsList = new ArrayList<>();
-        ensemblRestRegionsList.addAll(this.genomicToEnsemblRestRegionMap(genomicLocations).keySet());
+        //ensemblRestRegionsList.addAll(this.genomicToEnsemblRestRegionMap(genomicLocations).keySet());
```

Comment:
let's remove this comment

Reply:
marked as resolved, but I still see the commented out line not removed